.github/workflows/build.yml

```diff
@@ -0,0 +1,71 @@
+# ---------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ---------------------------------------------------------------------------
+
+name: build_jdk11
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
oh and I almost forgot. it does target the `main` branch, not `master`! I think renaming might make sense, also to keep the repository architecture up to date with citrusframework/citrus.

Reply:
and the new actions "in action": https://github.com/bbortt/citrus-simulator/actions.

Reply:
yes, I will rename the branch from master to main after this PR got merged. Also I think moving the ci to github is worth even if the workflow is failing at the moment with same error as in travis-ci